mkosi/resources/mkosi-addon/mkosi.conf

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[Distribution]
+Distribution=custom
+
+[Output]
+Output=initrd.addon
+Format=addon
+ManifestFormat=
+SplitArtifacts=
+
+[Content]
+Bootable=no
+MakeInitrd=yes
```

Comment:
I'm confident we don't need this, this creates the /init and /etc/initrd-release symlinks which shouldn't be in the addon.